proto/v1/hello_service.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package bytebase.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/wrappers.proto";
+
+option go_package = "generated-go/v1";
+
+service GreeterService {
+  rpc Hello(HelloRequest) returns (HelloResponse) {
+    option (google.api.http) = {
+      get: "/v2/hello/{name}"
```

Comment:
Is it a typo? or our v1 is protobuf and v2 is grpc?

Reply:
Got it in PR description.

Reply:
We already have v1 API. Need to use v2 for now till full migration to grpc.